db/migrate/20180409120422_create_physical_network_ports.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePhysicalNetworkPorts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :physical_network_ports do |t|
```

Comment:
Can this be just `network_ports` instead of physical_?

Reply:
`network_ports` table already exist, but seems like that its semantics for the system is not what we are looking for, once it has so many relationships, maybe you can tell it more accurately. The intention of this new table is really keep data about the physical ports (connections), so I think that the `physical_` prefix is required to make more clear this table semantic.